README.md

```diff
@@ -80,6 +81,12 @@ Updated & performant JavaScript & Node.js SDK for the Binance REST APIs and WebS
 - [LLMs & AI](#use-with-llms--ai)
 - [Contributions & Thanks](#contributions--thanks)
 
+## Used by
+
+<a href="https://dependents.info/tiagosiebler/binance">
+  <img src="https://dependents.info/tiagosiebler/binance/image" />
+</a>
+
```

Comment:
Thanks for the changes! This doesn't seem to be loading, can you try the markdown image syntax that's also used for badges? Would also like for it to be moved just above the star history section, near the end:
https://github.com/tiagosiebler/binance?tab=readme-ov-file#star-history

Reply:
Does seem to lead for me:

<img width="700" height="224" alt="image" src="https://github.com/user-attachments/assets/8a242793-44e7-4ecd-80ca-c45981c18a74" />

Could you check [this](https://github.com/gouravkhunger/binance/tree/ci/dependents?tab=readme-ov-file#used-by) link?